misc-scripts/xref_mapping/XrefParser/MGI_Desc_Parser.pm

```diff
@@ -35,16 +36,14 @@ sub run {
   my $source_id    = $ref_arg->{source_id};
   my $species_id   = $ref_arg->{species_id};
   my $files        = $ref_arg->{files};
-  my $verbose      = $ref_arg->{verbose};
-  my $dbi          = $ref_arg->{dbi};
-  $dbi = $self->dbi unless defined $dbi;
+  my $verbose      = $ref_arg->{verbose} || 0;
+  my $dbi          = $ref_arg->{dbi} || $self->dbi;
```

Comment:
I find it safer in these kind or param parsing to check for definedness instead of truth.
this means using `//` instead of `||`. But just a suggestion and personal preference.